### PR TITLE
feat : Added devops category and links for its resources in subcategories.

### LINKED ISSUE #1371: [ADD] Add DevOps and its Resources
### Category Name

DevOps

### Subcategory Name

DevOps

### Additional Context

_No response_

### Checklist

- [X] I have checked the existing [issues](https://github.com/rupali-codes/LinksHub/issues?q=is%3Aissue+)
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I have checked that this category does not exist
- [X] I have checked that this subcategory does not exist
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by ighoshsubho**
## Fixes Issue

Closes #1371 

## Changes proposed

- Added `DevOps` category
- Added its subcategories
- Added resources for `DevOps` category

## Note to reviewers

- I have made all the necessary changes to the files
- If any changes needed, please let me know

### CODE REVIEW

File: database/data.ts [hunk 1]
```diff
@@ -49,6 +49,21 @@ export const sidebarData: ISidebar[] = [
       { name: 'videos', url: '/videos', resources: DB.videos },
     ],
   },
+  {
+    category: 'devops',
+    subcategory: [
+      { name: 'cicd', url: '/cicd', resources: DB.cicd },
```
Comment (rupali-codes) [inline database/data.ts:55]:
```suggestion
      { name: 'CI/CD', url: '/cicd', resources: DB.cicd },
```

File: database/data.ts [hunk 2]
```diff
@@ -49,6 +49,21 @@ export const sidebarData: ISidebar[] = [
...
+      { name: 'devopsLifecycle', url: '/devopsLifecycle', resources: DB.devopsLifecycle },
```
Comment (rupali-codes) [inline database/data.ts:56]:
```suggestion
      { name: 'DevOps Life Cycle', url: '/devopsLifecycle', resources: DB.devopsLifecycle },
```

File: database/data.ts [hunk 3]
```diff
@@ -49,6 +49,21 @@ export const sidebarData: ISidebar[] = [
...
+      { name: 'devopsMethodologies', url: '/devopsMethodologies', resources: DB.devopsMethodologies },
```
Comment (rupali-codes) [inline database/data.ts:57]:
```suggestion
      { name: 'DevOps Methodologies', url: '/devopsMethodologies', resources: DB.devopsMethodologies },
```

**Review by CBID2 — APPROVED** [review]
Hi @ighoshsubho! :) I made a few tweaks to your descriptions. 
Hi [Contributor]! :)  In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're doing great! :)

**Comment by ighoshsubho** [comment]
ok @CBID2, will do that from next contributions.

**Review by k-deepak04 — APPROVED** [review]

**Pull request merged**